tests/unit/test_config.py

```diff
@@ -272,27 +272,57 @@ def test_family_inheritance_and_quotes(self, mock_glbl_cfg):
     [
         (  # Lack of icp
             {
-                'cycling mode': loader.ISO8601_CYCLING_TYPE,
+                'cycling type': loader.ISO8601_CYCLING_TYPE,
```

Comment:
Strictly speaking, this ought to be moved out of the `scheduling_cfg` dict into its own parameter. But seeing as I think I wrote these tests, I can do that later.

Reply:
It's now `cycling mode` again, which belongs in the config dict.